site/docs/v1/tech/apis/_application-response-body.adoc

```diff
@@ -233,12 +240,20 @@ Whether or not the Role is a considered to be a super user role. This is a marke
 [field]#application.samlv2Configuration.audience# [type]#[String]# [since]#Available since 1.6.0#::
 The audience for the SAML response sent to back to the service provider from FusionAuth. Some service providers require different audience values than the `issuer` and this configuration option lets you change the `audience` in the response.
 
-[field]#application.samlv2Configuration.callbackURL# [type]#[String]# [since]#Available since 1.6.0#::
+[field]#application.samlv2Configuration.authorizedRedirectURLs# [type]#[Array<String>]# [since]#Available since 1.20.0#::
+A list of URLs for the callback (sometimes called the Assertion Consumer Service or ACS). This is a set of values where FusionAuth may send the browser after the user logs in via SAML.
+
+[field]#application.samlv2Configuration.callbackURL# [type]#[String]# [since]#Available since 1.6.0# [deprecated]#Deprecated#::
```

Comment:
Since this is in the response body, and also deprecated, we should add a deprecation note. Use the same format as above for adding a deprecation note to the description area. 

For example: 

> This field is preserved for backwards compatibility and may be removed in a future release. This is the first value found in the [field]#authorizedRedirectURLs# parameter.

Reply:
Added.